pkgs/by-name/pi/piano-rs/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  pkg-config,
+  alsa-lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "piano-rs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ritiek";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-qZeH9xXQPIOJ87mvLahnJB3DuEgLX0EAXPvECgxNlq0=";
+  };
+
+  cargoHash = "sha256-vDqfWXeQVEnMWMjhAG/A0afff7dWMoQejDZjcVlYBMQ=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+  ];
+
+  buildInputs = [
+    alsa-lib
+  ];
+
+  postInstall = ''
+    mkdir -p "$out"/lib
+    install -Dm644 ${alsa-lib.out}/lib/libasound* "$out"/lib/
```

Comment:
This is not needed. The fixup scripts provided by `stdenv` will automatically link these libraries with the binary. You can verify this by running `ldd result/bin/.piano-rs-wrapped`